docs/example_static_html.md

```diff
@@ -3,50 +3,176 @@ title: "Example: Plain Old Static HTML"
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-html', port_forwards=8000)
 ```
 
-We'll be looking at the [oneup](https://github.com/windmilleng/tilt/tree/master/integration/oneup) project
-in `integration/oneup`.
+The first line tells Tilt to build an image with the name `example-html-image`
+in the directory `.` (the current directory).
+
+The second line tells Tilt to load the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)
+yaml. The image name in the `docker_build` call must match the container `image`
+reference in the `example-html` Deployment.
+
+The last line configures port-forwarding so that your server is
+reachable at http://localhost:8000/. The resource name in the `k8s_resource` call
+must match the Deployment's `metadata.name` in `kubernetes.yaml`.
 
-In the oneup directory, run
+Try it! Run:
 
 ```
+git clone https://github.com/windmilleng/tilt-example-html
+cd tilt-example-html/0-base
 tilt up
 ```
 
-Your terminal will turn into a status box that lets you watch your server come up. When it's ready,
-you will see the status icon turn green. The logs in the botton pane will display
-"Serving oneup on container port 8000."
+Your terminal will turn into a status box that lets you watch your server come
+up. Tilt will open a browser view of the same status.
+
+When it's ready, you will see the status icon turn green. The logs in the
+botton pane will display "Serving files on port 8000."
+
+[![Step 0](assets/docimg/example-static-html-image-1.png)](https://cloud.tilt.dev/snapshot/AejkyuULr2AjWu50Eck=){:.is-image}
 
-<div class="block u-margin1_5">
- <img src="assets/img/oneup.png">
-</div>
+## Step 1: Let's Add Benchmark Trickery
 
-Type `b` to open `oneup` in a browser window.
-Your browser will open `http://localhost:8100`.
-You should see the text `🍄 One-Up! 🍄`.
+Before we try to make this faster, let's measure it.
 
-Congratulations! You've run your first server with `tilt`.
+Tilt has escape hatches for running commands locally, so that you can integrate your existing scripts. 
 
-Type `ctrl-C` to quit the status box. When you're finished, run
+In this example, we use [`local_resource`](local_resource.html), which lets you
+trigger local jobs, or run local servers. We add a `local_resource` to our
+[Tiltfile](https://github.com/windmilleng/tilt-example-html/blob/master/1-measured/Tiltfile)
+that records when an update starts. We've also modified our server itself to
+read that start time and print the time elapsed.
 
+```python
+k8s_resource('example-html', port_forwards=8000, resource_deps=['deploy'])
+
+# Records the time from a code change to a new process.
+# Normally, you would let Tilt do deploys automatically, but this
+# shows you how to set up a custom workflow that measures it.
+local_resource(
+  'deploy',
+  'date +%s > start-time.txt')
 ```
-tilt down
+
+The `local_resource()` call creates a local resource named `deploy`. The second
```

Comment:
could cut out this line and notate this in code, if you want--seems a bit more concise to me:
```
local_resource(
  'deploy',  # name o the local resource
  'date %s...'  # command to run
)

Reply:
The problem with inline comments is that they end up wrapping weirdly on mobile